src/Page/Home.elm

```diff
@@ -54,139 +38,71 @@ init session =
             , div [ class "fs-5 mt-3 mb-5" ]
-                [ """Ecobalyse permet de comprendre et de calculer le coût environnemental des produits
-                     distribués en France. La méthodologie présentée est soumise à [concertation]({url})."""
-                    |> String.replace "{url}" Env.communityUrl
+                [ """Ecobalyse permet de comprendre et d’exprimer les impacts environnementaux des produits distribués en France
+                par le calcul d’un coût en points d’impact\u{202F}: **le coût environnemental**.
+                Découvrez nos outils et notre périmètre d’action\u{202F}!"""
                     |> Markdown.simple []
                 ]
             , div [ class "d-flex flex-column flex-sm-row gap-3 mb-4" ]
-                [ if enabledSections.food then
-                    button [ class "btn btn-lg btn-primary", onClick OpenCalculatorPickerModal ]
-                        [ text "Lancer le calculateur" ]
+                [ simulatorButton
+                    "Calculer l’impact d’un vêtement"
+                    Nothing
+                    True
+                    Route.TextileSimulatorHome
+                , if enabledSections.food then
+                    simulatorButton
+                        "Calculer l’impact de l’alimentation"
+                        (Just "Méthodologie en concertation")
+                        False
+                        Route.FoodBuilderHome
 
                   else
-                    a [ class "btn btn-lg btn-primary", Route.href Route.TextileSimulatorHome ]
-                        [ text "Lancer le calculateur" ]
-                , button
-                    [ class "btn btn-lg btn-outline-primary"
-                    , onClick OpenPresentationVideoModal
-                    ]
-                    [ text "Découvrir en vidéo" ]
-                , button
-                    [ class "btn btn-lg btn-outline-primary"
-                    , onClick <| ScrollIntoView "decouvrir-ecobalyse"
-                    ]
-                    [ text "En savoir plus" ]
-                ]
-            ]
-        , case modal of
-            CalculatorPickerModal ->
-                ModalView.view
-                    { size = ModalView.Large
-                    , close = CloseModal
-                    , noOp = NoOp
-                    , title = "Sélectionnez le secteur concerné"
-                    , subTitle = Nothing
-                    , formAction = Nothing
-                    , content = [ calculatorPickerModalContent ]
-                    , footer = []
-                    }
+                    text ""
+                , if enabledSections.objects then
+                    simulatorButton
+                        "Calculer l’impact d’un objet"
+                        (Just "Simulateur en construction")
+                        False
+                        (Route.ObjectSimulatorHome Scope.Object)
```

Comment:
Nit: it would be probably more explicit and readable to use a record here, eg.

```elm
simulatorButton
    { label = "Calculer l’impact d’un objet"
    , subLabel = (Just "Simulateur en construction")
    , callToAction = False
    , route = Route.ObjectSimulatorHome Scope.Object
    }
```